tests/links_tests/update_tests/test_film_update.py

```diff
@@ -0,0 +1,144 @@
+from chainer import cuda
```

Comment:
can you follow test_gin_update.py instead of ggnn, because sparse matmul is not tested.